### ISSUE TITLE
onnx opset_version=11 support

### ISSUE THREAD

**Issue by leehangyu** [KhronosGroup/NNEF-Tools#110]
Hi

There was a problem converting a upsample bilinear layer before opset_version 11 in onnx,

But it had been fixed it by version 11,  

I was wondering if you are planning to support onnx opset version 11

Thanks

**Comment by gyenesvi** [comment]
Can you please bit a bit more precise about what the problem is?
Where was the problem previously? In ONNX?
The ONNX to NNEF converter does support bilinear upsampling.
By the way, as I see from the ONNX documentation, the upsample operator has been deprecated in opset version 10. What changed in version 11?

**Comment by leehangyu** [comment]
Sorry for not being specific.

Let me show you how they are different.

This shows the result of upsampling bilinear in opset_version_9
![opset_version_9](https://user-images.githubusercontent.com/18110700/70486075-f0798800-1b33-11ea-92d8-c3d55d872240.PNG)

This shows the result of upsampling bilinear in opset_version_10
![opset_version_10](https://user-images.githubusercontent.com/18110700/70486110-05eeb200-1b34-11ea-8866-690f59f0d733.PNG)

This shows the result of upsampling bilinear in opset_version_11
![opset_version_11](https://user-images.githubusercontent.com/18110700/70486119-0ab36600-1b34-11ea-938e-aab9234195ce.PNG)

as you can see, only opset_version_11 returns correct result.

as of the fact that they are not the same, my model converted with opset_version_9 causes a critical difference between original and converted one.

**Comment by gyenesvi** [comment]
To me, this seems to be an incompatibility issue between PyTorch and ONNX (runtime).
What is the problem related to NNEF? Is there any error when you try to convert it to NNEF?

**Comment by leehangyu** [comment]
The only version I can convert it with is opset version 11, but NNEF does not support 11.

That's why I am asking for when NNEF supports opset version 11, there was warning show that NNEF does not support opset version 11? So I am wondering If there is any problem doing it with NNEF

**Comment by leehangyu** [comment]
Oh, by the way I figured out I do not need to interrupt the intermediate stage unless layers have weights, since upsampling layer has no weight I do not need to change the algorithm but pass the parameters.

I am sorry I didn't know it, I thought I had to correct exact match even intermediate stages but It is not.

The only upsampling layer I need to change is upsampling bilinear in TF-LITE model.

Thank you for reply and help.
